### PR TITLE
HUE-9383 potential Script execution in Share Document UI

### PR DESCRIPTION
**Pull request by Akhilsnaik**
## What changes were proposed in this pull request?

HUE-9383 potential Script execution in Share Document UI

## How was this patch tested?

Manually Tested for script Injection Resolution in Google Chrome, Firefox and Safari

### CODE REVIEW

**Comment by Akhilsnaik** [comment]
@JohanAhlen  @romainr  Please help me review this.

File: desktop/core/src/desktop/templates/common_share.mako [hunk 1]
```diff
@@ -21,7 +21,7 @@ from django.utils.translation import ugettext as _
   <a>
     <div>
       <i data-bind="css: icon"></i>
-      <span data-bind="html: label"></span>
+      <span data-bind="text: label"></span>
```
Comment (JohanAhlen) [inline desktop/core/src/desktop/templates/common_share.mako:24]: The html binding should already be sanitized, see [ko.html.js](https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/js/ko/bindings/ko.html.js)

Comment (romainr) [reply desktop/core/src/desktop/templates/common_share.mako:24]: +1, maybe you were on an old Hue (can check the history of ko.html.js)

Comment (Akhilsnaik) [reply desktop/core/src/desktop/templates/common_share.mako:24]: Hi @JohanAhlen ,

its true that already the xss attack is been taken care at ko.html.js  , But ideally the label should be displayed as text .

you can see further implementations of how users are handled as as Text : https://github.com/cloudera/hue/blob/dc67dc6ad4df0a2db0f16ef9216fb4bf87045e67/desktop/core/src/desktop/templates/common_share.mako#L48

https://github.com/cloudera/hue/blob/dc67dc6ad4df0a2db0f16ef9216fb4bf87045e67/desktop/core/src/desktop/templates/common_share.mako#L58

**Pull request merged**